service/http.proxy/src/main/java/org/kaazing/gateway/service/http/proxy/HttpProxyServiceHandler.java

```diff
@@ -152,6 +154,25 @@ public void operationComplete(ConnectFuture future) {
             }
         }
 
+        /**
+         * Helper method performing loop detection
+         * @param connectSession
```

Comment:
Either add javadoc for parameters or remove the @param (My IDE highlights it  and doesn't give the file a green check !)